hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/utils/FederationStateStoreFacade.java

```diff
@@ -513,6 +528,26 @@ public Map<String, SubClusterPolicyConfiguration> invoke(
     return cacheRequest;
   }
 
+  private Object buildGetApplicationHomeSubClusterRequest(ApplicationId applicationId) {
+    final String cacheKey = buildCacheKey(getClass().getSimpleName(),
+        GET_APPLICATION_HOME_SUBCLUSTER_CACHEID, applicationId.toString());
+    CacheRequest<String, SubClusterId> cacheRequest =
+        new CacheRequest<>(
+            cacheKey,
+            input -> {
+              GetApplicationHomeSubClusterResponse response =
+                  stateStore.getApplicationHomeSubCluster(
+                      GetApplicationHomeSubClusterRequest.newInstance(applicationId));
```

Comment:
Extract the request and probably do indentation like:
```
    CacheRequest<String, SubClusterId> cacheRequest = new CacheRequest<>(
        cacheKey,
```

Reply:
I will fix it.